torch/onnx/utils.py

```diff
@@ -1240,7 +1238,7 @@ def get_ns_op_name_from_custom_op(symbolic_name):
                            alphanumerical characters"
                            .format(symbolic_name))
     ns, op_name = symbolic_name.split("::")
-    unaccepted_domain_names = ["onnx", "aten", "prim"]
+    unaccepted_domain_names = ["onnx", "aten"]
     if ns in unaccepted_domain_names:
```

Comment:
I'd rewrite this to:
`if ns in ("aten", "onnx"):`

* No need for a local variable that is used only once.
* A tuple is better than a list for an immutable sequence, like this one.